src/plugins/recordTypes/collectionobject/forms/inventory.jsx

```diff
@@ -0,0 +1,132 @@
+            </Field>
+
+            <Field name="inventoryStatusList">
+              <Field name="inventoryStatus" />
+            </Field>
+          </Col>
+
+          <Col>
+            <Field name="briefDescriptions">
+              <Field name="briefDescription" />
+            </Field>
+          </Col>
+        </Row>
+
+        <Field name="titleGroupList">
+          <Field name="titleGroup">
+            <Panel>
+              <Row>
+                <Col>
+                  <Field name="title" />
+                  <Field name="titleLanguage" />
+                </Col>
+
+                <Col>
+                  <Field name="titleType" />
+
+                  <Field name="titleTranslationSubGroupList">
+                    <Field name="titleTranslationSubGroup">
+                      <Field name="titleTranslation" />
+                      <Field name="titleTranslationLanguage" />
+                    </Field>
+                  </Field>
+                </Col>
+              </Row>
+            </Panel>
+          </Field>
+        </Field>
+
+        <Field name="objectNameList">
+          <Field name="objectNameGroup">
+            <Field name="objectName" />
+            <Field name="objectNameCurrency" />
+            <Field name="objectNameLevel" />
+            <Field name="objectNameSystem" />
+            <Field name="objectNameType" />
+            <Field name="objectNameLanguage" />
+            <Field name="objectNameNote" />
+          </Field>
+        </Field>
+      </Panel>
+
+      <Panel name="desc" collapsible collapsed>
+        {extensions.dimension.form}
+      </Panel>
+
+      <Panel name="prod" collapsible collapsed>
+        <Row>
+          <Col>
+            <Field name="objectProductionDateGroupList">
+              <Field name="objectProductionDateGroup" />
+            </Field>
+
+            <Field name="objectProductionPersonGroupList">
+              <Field name="objectProductionPersonGroup">
+                <Field name="objectProductionPerson" />
+                <Field name="objectProductionPersonRole" />
+              </Field>
+            </Field>
+          </Col>
+
+          <Col>
+            <Field name="objectProductionOrganizationGroupList">
+              <Field name="objectProductionOrganizationGroup">
+                <Field name="objectProductionOrganization" />
+                <Field name="objectProductionOrganizationRole" />
+              </Field>
+            </Field>
+          </Col>
+        </Row>
+      </Panel>
+
+      <Panel name="hierarchy" collapsible collapsed>
+        <Field name="relation-list-item" subpath="rel:relations-common-list" />
+      </Panel>
+    </Field>
+  );
+};
+
+export default configContext => ({
+  disabled: true,
+  messages: defineMessages({
+    name: {
+      id: 'form.collectionobject.inventory.name',
+      defaultMessage: 'Inventory Template',
+    },
+  }),
+  sortOrder: 1,
+  template: template(configContext),
+});
```

Comment:
If all we're doing is hiding this form, it's enough to add `disabled: true`. Don't need to reiterate messages, sortOrder, and template. And `disabled: true` is short enough that I don't think it's worth breaking out into a separate file, so I would put it directly in forms/index.js.